### ISSUE TITLE
Markdown plugin: sometimes links aren't working

### ISSUE THREAD

**Issue by ArthurHoaro** [shaarli/Shaarli#672]
If you copy/paste Shaarli's `README.md` into shaarli's link description, blazy lib link isn't working.

I don't know if it's related to the markdown plugin or Parsedown, but the markdown seems correct:

```
### Goodies
- thumbnail generation for images and video services:
dailymotion, flickr, imageshack, imgur, vimeo, xkcd, youtube...
    - lazy-loading with [bLazy](http://dinbror.dk/blazy/)
- [PubSubHubbub](https://code.google.com/p/pubsubhubbub/) protocol support
- URL cleanup: automatic removal of `?utm_source=...`, `fb=...`
- discreet pop-up notification when a new release is available
```

result:

    <li>lazy-loading with [bLazy](<a href="http://dinbror.dk/blazy/"><a href="http://dinbror.dk/blazy/">http://dinbror.dk/blazy/</a></a>)</li>

For example, Github render:

### Goodies
- thumbnail generation for images and video services:
dailymotion, flickr, imageshack, imgur, vimeo, xkcd, youtube...
    - lazy-loading with [bLazy](http://dinbror.dk/blazy/)
- [PubSubHubbub](https://code.google.com/p/pubsubhubbub/) protocol support
- URL cleanup: automatic removal of `?utm_source=...`, `fb=...`
- discreet pop-up notification when a new release is available

**Comment by virtualtam** [comment]
The issue doesn't happen on http://parsedown.org/demo
I would have suggested upgrading, yet we're already using the latest released version (1.6.0) - https://github.com/erusev/parsedown/releases

I wasn't able to find a related change from https://github.com/erusev/parsedown/compare/1.6.0...master

**Comment by ArthurHoaro** [comment]
I'll submit a PR to import it with Composer, it'll be easier to follow versions.
I found the issue. `reverse_text2clickable` remove Shaarli automatic links, except in code blocks such as "```", and lines starting with 4 spaces. I didn't think of nested lists.
